src/python/WMCore/MicroService/Tools/Common.py

```diff
@@ -25,12 +25,34 @@
 from WMCore.Services.pycurl_manager import RequestHandler
 from WMCore.Services.pycurl_manager import getdata as multi_getdata
 
+# DBS agregators
+from dbs.apis.dbsClient import aggRuns, aggFileLumis
+
 
 # static variables
 STEP_PAT = re.compile(r'Step[0-9]')
 TASK_PAT = re.compile(r'Task[0-9]')
 
 
+def isEmptyResults(row):
+    """
+    _isEmptyResults_
+
+    Evaluates whether row data contains empty result set
+    :return: bool
+    """
+    if 'data' not in row:
+        raise Exception("provided result dict does not contain 'data' key")
+    code = 200 # default HTTP status code
```

Comment:
Valentin, just a final suggestion that will make this code look better. How about replacing L46-L50 by:
```
# if code is not present in row it means it was success (HTTP status code 200)
code = int(row.get('code', 200))
```

Reply:
done